packages/flutter/lib/src/cupertino/text_theme.dart

```diff
@@ -47,97 +49,90 @@ const TextStyle _kDefaultTabLabelTextStyle = TextStyle(
 );
 
-// Eyeballed value since it's not documented in https://developer.apple.com/design/resources/.
+// Please update _DefaultCupertinoTextThemeData and _DefaultCupertinoTextThemeData
+// accordingly after changing the default color here, as their implementation
+// depends on the default value of the color field.
+//
 // Inspected on iOS 13 simulator with "Debug View Hierarchy".
-const TextStyle _kDefaultDateTimePickerLightTextStyle = TextStyle(
+// Value extracted from off-center labels. Centered labels have a font size of 25pt.
+const TextStyle _kDefaultDateTimePickerTextStyle = TextStyle(
   inherit: false,
   fontFamily: '.SF Pro Display',
   fontSize: 21,
   fontWeight: FontWeight.normal,
-  color: CupertinoColors.black,
+  color: CupertinoColors.label,
 );
 
-// Eyeballed value since it's not documented in https://developer.apple.com/design/resources/.
-// Inspected on iOS 13 simulator with "Debug View Hierarchy".
-const TextStyle _kDefaultDateTimePickerDarkTextStyle = TextStyle(
-  inherit: false,
-  fontFamily: '.SF Pro Display',
-  fontSize: 21,
-  fontWeight: FontWeight.normal,
-  color: CupertinoColors.white,
-);
+TextStyle _resolveTextStyle(TextStyle style, BuildContext context, bool nullOk) {
+  // This does not change the shadow color, etc.
+  Paint foreground, background;
+  if (style?.foreground != null)
+    foreground..color = CupertinoDynamicColor.resolve(style?.foreground?.color, context, nullOk: nullOk);
+  if (style?.background != null)
+    background..color = CupertinoDynamicColor.resolve(style?.background?.color, context, nullOk: nullOk);
+
+  return style?.copyWith(
+    color: CupertinoDynamicColor.resolve(style?.color, context, nullOk: nullOk),
+    backgroundColor: CupertinoDynamicColor.resolve(style?.backgroundColor, context, nullOk: nullOk),
+    decorationColor: CupertinoDynamicColor.resolve(style?.decorationColor, context, nullOk: nullOk),
+    foreground: foreground,
+    background: background,
+  );
+}
 
 /// Cupertino typography theme in a [CupertinoThemeData].
 @immutable
 class CupertinoTextThemeData extends Diagnosticable {
   /// Create a [CupertinoTextThemeData].
   ///
-  /// The [primaryColor] and [isLight] parameters are used to derive TextStyle
-  /// defaults of other attributes such as [textStyle] and [actionTextStyle]
-  /// etc. The default value of [primaryColor] is [CupertinoColors.activeBlue]
-  /// and the default value of [isLight] is true.
+  /// The [primaryColor] is used to derive TextStyle defaults of other attributes
+  /// such as [textStyle] and [actionTextStyle] etc. The default value of [primaryColor]
+  /// is [CupertinoColors.activeBlue].
   ///
   /// Other [TextStyle] parameters default to default iOS text styles when
   /// unspecified.
   const CupertinoTextThemeData({
-    Color primaryColor,
-    Brightness brightness,
+    Color primaryColor = CupertinoColors.systemBlue,
+    @deprecated Brightness brightness, //ignore: avoid_unused_constructor_parameters
```

Comment:
Breaking change. `brightness` is no longer used anywhere and would love to get rid of it in the near future.

Reply:
per our style guide: Please add a comment to all `// ignore:` explaining why the ignore is there and when (if at all) it can be removed.